shared/constants/fs.js

```diff
@@ -21,6 +21,7 @@ import flags from '../util/feature-flags'
 
 export const sendLinkToChatFindConversationWaitingKey = 'fs:sendLinkToChatFindConversation'
 export const sendLinkToChatSendWaitingKey = 'fs:sendLinkToChatSend'
+export const deleteFolderWaitingKey = (path: Types.Path) => `fs:deleteFolder:${Types.pathToString(path)}`
```

Comment:
There's only one confirm modal shown at a time, so we can just use one single waitingKey here. Then it'll be a constant rather than a function.